example/simple_testing/kotlin/src/main/kotlin/dev/diplomattest/simple_testing/Main.kt

```diff
@@ -0,0 +1,128 @@
+package dev.diplomattest.simple_testing;
+
+import com.sun.jna.Callback
+import com.sun.jna.Library
+import com.sun.jna.Native
+import com.sun.jna.Pointer
+import com.sun.jna.Structure
+
+// ------------------------------------------------------------------------------------------- Generated code
+
+class DiplomatTrait_TesterTrait_VTable_Native: Structure(), Structure.ByValue {
+    @JvmField
+    internal var run_test_trait_fn_callback: DiplomatCallback_Lib.Runner_DiplomatTraitMethod_TesterTrait_test_trait_fn
+        = object :  DiplomatCallback_Lib.Runner_DiplomatTraitMethod_TesterTrait_test_trait_fn {
+                override fun invoke(ignored: Pointer?, input: Int): Int {
+                    throw Exception("ERROR NOT IMPLEMENTED")
+                }
+            }
+    @JvmField
+    internal var run_test_void_trait_fn_callback: DiplomatCallback_Lib.Runner_DiplomatTraitMethod_TesterTrait_test_void_trait_fn
+        = object :  DiplomatCallback_Lib.Runner_DiplomatTraitMethod_TesterTrait_test_void_trait_fn {
+                override fun invoke(ignored: Pointer?): Unit {
+                    throw Exception("ERROR NOT IMPLEMENTED")
+                }
+            }
+    @JvmField
+    internal var destructor: Pointer = Pointer(0L);
+  
+    // Define the fields of the struct
+    override fun getFieldOrder(): List<String> {
+        return listOf("run_test_trait_fn_callback", "run_test_void_trait_fn_callback", "destructor")
+    }
+}
+
+class DiplomatTrait_TesterTrait_Wrapper_Native: Structure(), Structure.ByValue {
+    @JvmField
+    internal var data_: Pointer = Pointer(0L);
+    @JvmField
+    internal var vtable: DiplomatTrait_TesterTrait_VTable_Native
+        = DiplomatTrait_TesterTrait_VTable_Native();
+  
+    // Define the fields of the struct
+    override fun getFieldOrder(): List<String> {
+        return listOf("data_", "vtable")
+    }
+}
+
+class DiplomatCallback_TesterTrait_Wrapper internal constructor (
```

Comment:
I'd expect something like this
```suggestion
class Wrapper internal constructor (
```